assessments/0009-iin-toto.md

```diff
@@ -0,0 +1,292 @@
+
+**Content maintainability**
+
+Almost all doc content is in GitHub (in README files, the Specification, or comments in demos). Most of it should be surfaced to indexed documents on the website to make it versionable, editable, and localizable.
+- We recommend moving most of the documentation into read-the-docs (RTD) so that it can properly indexed and maintained.
+
+High-level overviews are provided in several places, and mostly embedded in other documents. These are hard to compare and keep in sync. 
+- Overviews with increasing depth should be offered explicitly to different audiences (evaluators, new users, experienced users, contributors)
+- A single source for some of the text would help. 
+
+**Content creation processes**
+
+- The [read-the-docs site](https://in-toto.readthedocs.io/en/latest/) has an [Edit the Doc button](https://github.com/in-toto/in-toto/blob/develop/doc/source/index.rst). Instead of pointing directly to the source files, it should point to a page with instructions for [doc contributors](https://github.com/in-toto/community/blob/main/CONTRIBUTING.md), and a link to the [governance policies](https://github.com/in-toto/community/blob/main/GOVERNANCE.md).
+
+Material explicitly addressed to documentation contributions and standards should be added, and accessible from the top-level doc roadmap as well as from the existing contributor pages.
+- Procedures for creating and maintaining docs need to be documented.
+   - The process should include a policy for where reference docs live in the repo structure and how docs for different implementations are distinguished and identified.
+- Policy and procedures for documenting implementations need to be decided and published for contributors.
+   - Reference documentation for additional implementations should be actively solicited from the contributor community.
+
+## Contributor documentation
+
+| Criteria                                  | 1   | 2   | 3   | 4   | 5   |
+| ---                                       | --- | --- | --- | --- | --- |
+| Communication methods documented          |     |     |     |  x  |     |
+| Beginner friendly issue backlog           |     |  x  |     |     |     |
+| “New contributor” getting started content |     |  x  |     |     |     |
+| Project governance documentation          |     |     |     |  x  |     |
+
+Scale:
+
+- 1 = (Is not present or requires significant work)
+- 3 = (Is present, but needs work)
+- 5 = (Is executed extremely well or no improvement required)
+
+**Comments**
+
+- **Communication methods documented:** [Contact channels list](https://in-toto.io/contact/) available from home page under **Community > Contact**.
+
+- **Beginner friendly issue backlog:** "Good First Issue" label is available and used for PRs for specific implementations. Not in much general use.
```

Comment:
We should take this opportunity to surface a new tool we've built: https://clotributor.dev/, https://github.com/cncf/clotributor

"One of the CLOTributor's goals is to surface interesting opportunities for potential contributors to Cloud Native projects, allowing them to find those that suit their skills and interests best.

To achieve this, CLOTributor scans periodically hundreds of repositories, indexing issues that match certain criteria:

    Contain the help wanted label
    Their state is OPEN
    They are unassigned
    Updated within the last year
"